arrows/core/metadata_map_io_csv.cxx

```diff
@@ -222,7 +230,8 @@ bool
 metadata_map_io_csv
 ::check_configuration( VITAL_UNUSED vital::config_block_sptr config ) const
```

Comment:
Could the comment in the header file be expanded to explain what this function checks for?